.github/workflows/release.yaml

```diff
@@ -0,0 +1,116 @@
+name: Release
```

Comment:
Hmm, I'm not quite sure about this structure/naming. Since one can trigger this "Release" workflow with `release` set to `false`, I think it is a little bit misleading. Also, if I'm not mistaken, all jobs except `build` are skipped if `release` is set to `false`. Can't we also allow manual triggering of the `build-and-test` workflow (equivalent to running this workflow without enable `release`), and remove the `release` input from here and assume it's always an actual release?

Reply:
@8R0WNI3 : you are correct (actually, I think I felt the urge of adding a comment hinting towards removal).

My idea for leaving it in, and setting default-value to `false` was two-fold:

for once, this might be an option to trigger a test-release-run (admittedly of limited value, as you pointed out)
secondly, I assumed it might be a nice "are-you-sure"-thingy avoiding accidental triggering. OTOH reconsidering, I think that might not actually be an issue. will change it as suggested.